bin/yarn-install

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+require('shelljs/global')
+
+exec('cd cli && yarn && cd ..')
```

Comment:
Let's nudge the user into the right direction, and try to use yarn as much as possible everywhere. Ideally, all of Exosphere only uses yarn.